mobile-first/css/styles.css

```diff
@@ -0,0 +1,449 @@
+  width: 500px;
+  display: flex;
+}
+
+.container .navcontainer .navmenu a{
+  padding: 20px 0 20px 0;
+  text-align: center;
+  border-top: 1px solid black;
+  font-size: 1.3em;
+}
+
+#menu-links {
+  display: none;
+}
+
+.last-menu {
+  border-bottom: 1px solid black;
+}
+
+.navmenu a {
+	
+	text-decoration: none;
+	color: grey;
+	
+}
+
+.navmenu a:hover {
+	text-decoration: underline;
+
+}
+
+/* header */
+
+.container .headermain img {
+  display: none;
+}
+
+
+.headermain {
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	padding: 80px 0 32px 0;
+	margin: 80px 0 43.2px 0;
+}
+
+.headertext {
+	flex: 0;
+	text-align: center;
+	display: flex;
+	flex-direction: column;
+  align-items: center;
+}
+
+.headertext h1 {
+	font-size: 72px;
+}
+
+.home-page .headermain .headertext .get-started {
+	border: 1px solid black;
+	padding: 6.4px 16px;
+	width: 169.6px;
+	cursor: pointer;
+	text-align: center;
+	font-size: 1.1rem;
+	margin-top: 6.4px;
+}
+
+.home-page .headermain .headertext .get-started:hover {
+	background-color: black;
+	color: white;
+}
+
+.headermain .logo {
+	flex: 0;
+}
+
+/* above image content */
+
+.container .above-img {
+  display: flex;
+  flex-direction: column;
+  padding: 52px 20px 0 20px;
+}
+
+.container .above-img .featuress {
+  padding-bottom: 46px;
+}
+
+.container .above-img .about {
+  padding-bottom: 53px;
+}
+
+/* middle image  */
+
+.container .middle-img {
+  padding: 0 20px 30px 20px;
+  width: 100%;
+}
+
+/* below image content */
+
+.container .below-img {
+  padding: 0 20px;
+}
+
+.container .below-img .product {
+    padding: 63px 0 0 0;
+}
+
+.container .below-img .vision {
+  padding: 64px 0 30px 0;
+}
+
+.container address {
+  padding: 25px 20px 0 20px;
+}
+
+footer {
+  padding-top: 25px;
+  text-align: center;
+}
+
+@media (min-width: 800px) {
+  body {
+    margin: 0 auto;
+    max-width: 800px;
+    display: block;
+    position: relative;
+  }
+
+  .container nav {
+    width: 800px;
+    max-width: 880px;
+  }
+
+  .container .navcontainer .logo {
+    padding: 0;
+  }
+
+  .container .navcontainer .navmenu {
+    display: flex;
+    flex-direction: row;
+    padding-left: 30px;
+    justify-content: space-between;
+    width: 92%;
+  }
+
+  .container .navcontainer .navmenu a {
+    border: none;
+    font-size: 1em;
+  }
+
+  #menu-links {
+    display: flex;
+  }
+
+  .container .navcontainer .burger {
+    display: none;
+  }
+
+  /* above image content*/
+
+  .container .above-img {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    padding: 28px 30px 5px 30px;
+  }
+
+  .container .above-img .above-content-block {
+    width: 50%;
+    padding-bottom: 0;
+  }
+  
+  .container .above-img .featuress {
+    padding-bottom: 46px;
+  }
+  
+  .container .above-img .about {
+    padding-left: 10px;
+    padding-bottom: 0;
+  }
+
+  .container .middle-img {
+
+  }
+  
+  /* below image content */
+  .container .below-img {
+    padding: 0 0 0 0;
```

Comment:
Why is this here if it's all zeros?

Reply:
i meant to do padding :none; it's to reset past code from the previous media queries